src/backend/config.ts

```diff
@@ -393,24 +393,17 @@ abstract class GlobalConfig {
   }
 
   /**
-   * Checks if a Wine version has Wineboot/Wineserver executables and returns the path to those if they're present
+   * Checks if a Wine version has the Wineserver executable and returns the path to those if they're present
    * @param wineBin The unquoted path to the Wine binary ('wine')
-   * @returns The quoted paths to wineboot and wineserver, if present
+   * @returns The quoted paths to wineserver, if present
    */
-  public getWineExecs(wineBin: string): {
-    wineboot: string
-    wineserver: string
-  } {
+  public getWineExecs(wineBin: string): { wineserver: string } {
```

Comment:
Somewhat unnecessary now to return an object with just one key
Might as well just return the path directly

Reply:
this required more code changes, and I just found it simpler to leave it the way it is, but I can it change if needed